packages/chakra-ui-docs/docs/avatar.mdx

```diff
@@ -1 +1,150 @@
+
+<br />
+<LiveEditor>
+{`
+<Stack isInline>
+    <Avatar name="Oshigaki Kisame" src="https://bit.ly/broken-link" />
+    <Avatar name="Sasuke Uchiha" src="https://bit.ly/broken-link" />
+    <Avatar src="https://bit.ly/broken-link" />
+</Stack>
+`}
+</LiveEditor>
+
+
+## Avatar with badge
+
+In some products, you might need show a badge on the right corner of the avatar,
+what we call a **badge**. Here's an example that shows if the user is online:
+
+<br />
+<LiveEditor>
+{`
+<Stack isInline spacing={4}>
+    <Avatar>
+        <AvatarBadge size="1.25em" bg="green.500" />
+    </Avatar>
+    <!-- You can also change the borderColor and bg of the badge -->
+    <Avatar>
+        <AvatarBadge borderColor="papayawhip" bg="tomato" size="1.25em" />
+    </Avatar>
+</Stack>
+`}
+</LiveEditor>
+
+
+> Note the use of `em` for the size of the `AvatarBadge`. this is useful to size
+> the badge relative to the avatar font size
+
+## AvatarGroup
+
+In some cases, you might need to stack avatars as a group. Use the `AvatarGroup`
+component.
+
+- To limit the amount of avatars to show, use the `max` prop. It'll truncate the avatars and show a "+X" label (where X is the remaining avatars).
+- To size all the avatars equally, pass the size prop.
+- To adjust the spacing between the avatars, pass the `spacing` prop.
+
+<LiveEditor>
+{`
+<AvatarGroup size="md" max="2">
+    <Avatar name="Ryan Florence" src="https://bit.ly/ryan-florence" />
+    <Avatar name="Segun Adebayo" src="https://bit.ly/sage-adebayo" />
+    <Avatar name="Kent Dodds" src="https://bit.ly/kent-c-dodds" />
+    <Avatar name="Prosper Otemuyiwa" src="https://bit.ly/prosper-baba" />
+    <Avatar name="Christian Nwamba" src="https://bit.ly/code-beast" />
+</AvatarGroup>
+`}
+</LiveEditor>
+
+
+## Props
+
+The Avatar component composes `Box` component so you can pass props for `Box`.
+
+| Name       | Type                                       | Default | Description                                                                       |
+|------------|--------------------------------------------|---------|-----------------------------------------------------------------------------------|
+| size       | `2xs`, `xs`, `sm`, `md`, `lg`, `xl`, `2xl` | `md`    | The size of the avatar                                                            |
+| showBorder | `boolean`                                  | `false` | If `true`, the `Avatar` will show a border around it. Best for a group of avatars |
+| name       | `string`                                   |         | The name of the user in the avatar                                                |
+| src        | `string`                                   |         | The image url of the `Avatar`                                                     |
+| children   | `Vue.VNode`                                |         | The badge at the bottom right corner of the Avatar.                               |
```

Comment:
Vue components do now have a `children` prop unless declared. In Vue we use slots.